### PR TITLE
fixed parameter definition rente mit 63 plus notes.

### PR DESCRIPTION
**Pull request by TeBackh**
### What problem do you want to solve?

Reference the issue or discussion, if there is any. Provide a description and/or bullet
points to describe the changes in this PR.

### Todo
- [x] delete unnecessary code lines that lead to imprecision

### CODE REVIEW

File: src/_gettsim/parameters/ges_rente.yaml [hunk 1]
```diff
@@ -875,12 +875,12 @@ altersgrenze_für_frauen_vorzeitig:
...
 altersgrenze_langj_versicherte_vorzeitig:
   name:
-    de: Altersgrenze für vorzeitigen Bezug der Altersrente für langjährig Versicherte.
+    de: Altersgrenze für vorzeitigen Bezug der Altersrente für langjährig Versicherte
     en: Early retirement age for long term insured
   description:
     de: >-
-      Frühesmögliches Alter ab dem die Altersrente für langjährig Versicherte bezogen
-      werden kann (mit Abschlägen).
+      Frühesmögliches Alter ab dem die Altersrente für langjährig Versicherte be-
```
Comment (hmgaudecker) [inline src/_gettsim/parameters/ges_rente.yaml:882]: Please don't break words. In general, why all those unrelated changes to the texts?

File: src/_gettsim/parameters/ges_rente.yaml [hunk 2]
```diff
@@ -973,23 +976,16 @@ altersgrenze_besonders_langj_versicherte:
       upper_threshold: inf
       rate_linear: 0
       intercept_at_lower_threshold: 65
-  2014-07-01:
+  2014-07-01:  # See comment above *
```
Comment (hmgaudecker) [inline src/_gettsim/parameters/ges_rente.yaml:979]: Leave as is, no inline comments, please. 

Are we sure that the results are correct on, say, 1 August 2014? Just wondering about the granularity here since thresholds are based on years but introduction was at the monthly level.

**Review by hmgaudecker — COMMENTED** [review]
Thanks! 

As mentioned in #493, we should wait until we have the new machinery for time-varying functions in place. `_ges_rente_besond_langj_altersgrenze` should not exist before 2014-07-01.

File: src/_gettsim/parameters/ges_rente.yaml (hunk 2 above)
Comment (TeBackh) [reply src/_gettsim/parameters/ges_rente.yaml:979]: To calculate the eligibility for a given date it should work -  and I think this is the main function at the moment.
(For the details we should talk really , there is just too many layers of thinking right know to provide relevant answers in written comment.)

Comment (hmgaudecker) [reply src/_gettsim/parameters/ges_rente.yaml:979]: Great. Let's see about that once we have merged #495 and potentially other larger changes.

File: src/_gettsim/parameters/ges_rente.yaml (hunk 1 above)
Comment (ChristianZimpelmann) [reply src/_gettsim/parameters/ges_rente.yaml:882]: I guess pre-commit hooks complaining about line length. I will ask Laura to run pre-commit hooks over all files to avoid it in the future

**Review by ChristianZimpelmann — APPROVED** [review]
I agree with @hmgaudecker's comments. Otherwise, it is ready to merge.

Thanks!

**Comment by ChristianZimpelmann** [comment]
@hmgaudecker, why did you add the `blocked'-label. If I understand correctly, without this PR the new test cases in #428 are failing.

**Comment by hmgaudecker** [comment]
> @hmgaudecker, why did you add the `blocked'-label. If I understand correctly, without this PR the new test cases in #428 are failing.

Because I would like to see a proper solution for the introduction of the function, which will only be reasonable with the machinery from #495. That will not be part of the upcoming release.

If easier, please go ahead and merge, noting that point appropriately (I guess that was your idea behind #491?) so that it will not be forgotten.

**Review by ChristianZimpelmann — APPROVED** [review]
I added a note on `_ges_rente_besond_langj_altersgrenze` and addressed the last comment.

Ready to be merged after tests have run through.

Thanks @TeBackh and @hmgaudecker
